sympy/physics/mechanics/joint.py

```diff
@@ -40,7 +41,7 @@ class Joint(ABC):
         The parent body of joint.
     child : Body
         The child body of joint.
-    coordinates: iterable of dynamicsymbols, optional
+    coordinate : iterable of dynamicsymbols, optional
```

Comment:
Is this a typo here? Is it meant to be `coordinates` still but with the trailing space for formatting?

Reply:
Yep, fixed it